pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1451,6 +1451,21 @@ func (a *ApplicationBackupController) backupResources(
 
 	// Don't modify resources if mentioned explicitly in specs
 	resourceCollectorOpts := resourcecollector.Options{}
+	resourceCollectorOpts.ResourceCountLimit = k8sutils.DefaultResourceCountLimit
+	// Read configMap for any user provided value. this will be used in to List call of getResource eventually.
+	resourceCountLimitString, err := k8sutils.GetConfigValue(k8sutils.StorkControllerConfigMapName, metav1.NamespaceSystem, k8sutils.ResourceCountLimitKeyName)
```

Comment:
nit: resourceCountLimitString -> resourceCountLimit, string is not required

Reply:
that function returns the string , I need to do a  typecast . Moreover I have seen in stork in many places it is done the same way .. Hence keeping it the same way for now